services/app/src/hooks/useGameIdFromLocalStorage.js

```diff
@@ -0,0 +1,5 @@
+export const useGameIdFromLocalStorage = () => {

```

Comment:
I think the name is a bit too explicit, lets reduce it to `useGameId` because most likely we are not going to have any other kind of storage for gameId apart from `localStorage` so there's no real need for explicitly specifying that this hook is operating with `localStorage`.